test/src/test/java/cn/edu/tsinghua/iginx/integration/scaleout/IoTDBRestfulScaleOutIT.java

```diff
@@ -1,34 +1,77 @@
 package cn.edu.tsinghua.iginx.integration.scaleout;
```

Comment:
IoTDBRestfulScaleOutIT seems to have so similar functionality as IoTDBSqlScaleOutIt and IoTDBTagScaleOutIT. But why do they differ so much in their respective testing process and implementation?
Why not make a better abstraction for them to clear up the code?